google-account-plugin/resources/META-INF/plugin.xml

```diff
@@ -19,6 +19,8 @@
   <vendor>Google</vendor>
   <!-- "version" set by gradle-intellij-plugin -->
   <!-- "idea-version since-build" set by gradle-intellij-plugin -->
+  <!-- Workaround for gradle-intellij-plugin limitation: https://github.com/JetBrains/gradle-intellij-plugin/issues/66 -->
+  <idea-version since-build="143.381.42" />
```

Comment:
which version of IJ does that correspond to?  Can you include a link to the mapping table?